src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int MAX_NUMBER = 100;
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return colors[index].toString();
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants